<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coordinates Conversion Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/32x32/globe-icon.png</normaloff>:/icons/32x32/globe-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Convert to:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="toUtm">
        <property name="toolTip">
         <string>Conversion from Lon Lat coordinates to the UTM projection.</string>
        </property>
        <property name="text">
         <string>UTM Projection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="toLatLon">
        <property name="toolTip">
         <string>Conversion from UTM projection to Lon Lat geodesic coordinates.</string>
        </property>
        <property name="text">
         <string>Lon Lat</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="useSouthernHemisphere">
        <property name="toolTip">
         <string>Check this box to use Southern hemisphere, otherwise Northern will be used.</string>
        </property>
        <property name="text">
         <string>Southern</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Input:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="ddmmss">
        <property name="toolTip">
         <string>Check this box if the coordinates are formatted as DEGREES, MINUTES and SECONDS.</string>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="text">
         <string>Formatted as: DDÂº MM' SS''</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="inputText">
      <property name="toolTip">
       <string>Input coordinates: first x-axis coordinate (or longitude) then y-axis coordinate (or latitude), finally UTM Zone if necessary.</string>
      </property>
      <property name="whatsThis">
       <string>The input coordinates should be placed here.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="convert">
        <property name="toolTip">
         <string>Perform conversion</string>
        </property>
        <property name="text">
         <string>Convert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="toolTip">
         <string>Clear the text in both input and output text fields.</string>
        </property>
        <property name="text">
         <string>&amp;Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Output:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="outputText">
      <property name="toolTip">
       <string>The resulting coordinates: first x-axis coordinate (or longitude) then y-axis coordinate (or latitude), finally UTM Zone if necessary.</string>
      </property>
      <property name="whatsThis">
       <string>The output of the conversion will be here.</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <widget class="QMenu" name="menu_Datum">
     <property name="title">
      <string>&amp;Datum</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/16x16/world-go.png</normaloff>:/icons/16x16/world-go.png</iconset>
     </property>
     <addaction name="actionWGS84"/>
     <addaction name="actionHayford"/>
    </widget>
    <widget class="QMenu" name="menuDelimiter">
     <property name="title">
      <string>Delimiter</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/32x32/text-csv.png</normaloff>:/icons/32x32/text-csv.png</iconset>
     </property>
     <addaction name="actionTab"/>
     <addaction name="actionComma"/>
    </widget>
    <addaction name="menu_Datum"/>
    <addaction name="menuDelimiter"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveResults"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConvert"/>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionWGS84">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;WGS 84</string>
   </property>
  </action>
  <action name="actionHayford">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Hayford 1909</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/32x32/dialog-information.png</normaloff>:/icons/32x32/dialog-information.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionComma">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Comma</string>
   </property>
  </action>
  <action name="actionSpace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Space</string>
   </property>
  </action>
  <action name="actionTab">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tab</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/32x32/help.png</normaloff>:/icons/32x32/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="toolTip">
    <string>Help to use this program</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/32x32/document-open-3.png</normaloff>:/icons/32x32/document-open-3.png</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveResults">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/32x32/document-save-4.png</normaloff>:/icons/32x32/document-save-4.png</iconset>
   </property>
   <property name="text">
    <string>Save results</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/32x32/exit.png</normaloff>:/icons/32x32/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionConvert">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/32x32/run-rebuild.png</normaloff>:/icons/32x32/run-rebuild.png</iconset>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/32x32/edit-clear-locationbar-rtl.png</normaloff>:/icons/32x32/edit-clear-locationbar-rtl.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>toUtm</tabstop>
  <tabstop>toLatLon</tabstop>
  <tabstop>useSouthernHemisphere</tabstop>
  <tabstop>ddmmss</tabstop>
  <tabstop>inputText</tabstop>
  <tabstop>convert</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>outputText</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
